.editorconfig

```diff
@@ -48,7 +48,7 @@ end_of_line = crlf
 [AppRun]
 end_of_line = lf
 
-[CMakeLists.txt]
+[{CMakeLists.txt,CMakeSettings.json}]

```

Comment:
Hmm, last time I tried, multiple patterns are not supported by all the tools. hope this works in vs code